ConsistencyManager/ConsistencyManager.swift

```diff
@@ -383,6 +399,25 @@ open class ConsistencyManager {
     }
 
     /**
+     Adds an update listener to the consistency manager.
+     This listener will get notified of ALL changes posted to the consistency manager.
+     This method must be called on the main thread.
+     */
+    open func addModelUpdatesListener(_ updatesListener: ConsistencyManagerUpdatesListener) {
+        modelUpdatesListeners.append(updatesListener)
+    }
+
+    /**
+     Removes an update listener to the consistency manager.
+     This method must be called on the main thread.
+     */
+    open func removeModelUpdatesListener(_ updatesListener: ConsistencyManagerUpdatesListener) {
+        modelUpdatesListeners = modelUpdatesListeners.filter { currentListener in
+            currentListener === updatesListener
+        }
+    }
```

Comment:
It seems like this is saying it removes a single listener, but the implementation removes all listeners aside from the one passed in. So either the method name and documentation is not clear enough or the implementation is wrong.